pkg/dinghyfile/builder_test.go

```diff
@@ -1253,4 +1249,100 @@ pipelines:
 			}
 		})
 	}
-}
\ No newline at end of file
+}
+
+func TestPipelineBuilder_getContent(t *testing.T) {
+	type fields struct {
+		Downloader                  Downloader
+		Depman                      DependencyManager
+		TemplateRepo                string
+		TemplateOrg                 string
+		DinghyfileName              string
+		Client                      util.PlankClient
+		DeleteStalePipelines        bool
+		AutolockPipelines           string
+		EventClient                 events.EventClient
+		Parser                      Parser
+		Logger                      log.DinghyLog
+		Ums                         []Unmarshaller
+		Notifiers                   []notifiers.Notifier
+		PushRaw                     map[string]interface{}
+		GlobalVariablesMap          map[string]interface{}
+		RepositoryRawdataProcessing bool
+		RebuildingModules           bool
+		Action                      pipebuilder.BuilderAction
+	}
+	tests := []struct {
+		name   string
+		fields fields
+		want   map[string]interface{}
+	}{
+		{
+			name: "Content should return a map populated with 'raw' property if no Log Events are available",
+			fields: fields{
+				Logger: NewDinghylog(),
+				PushRaw: map[string]interface{}{
+					"head_commit": map[string]interface{}{
+						"id": "a5fc63bd5a8bdb342d1e83933a5b5c99010e61e4",
+					},
+				},
+			},
+			want: map[string]interface{}{
+				"raw": map[string]interface{}{
+					"head_commit": map[string]interface{}{
+						"id": "a5fc63bd5a8bdb342d1e83933a5b5c99010e61e4",
+					},
+				},
+			},
+		},
+		{
+			name: "Content should return a map populated with 'raw' and 'logevent' properties",
```

Comment:
Only github uses PushRaw AFAIK, so maybe we should add one extra test also when it's nil